src/models.rs

```diff
@@ -37,6 +44,9 @@ pub trait Model {
     /// The framebuffer size in pixels.
     const FRAMEBUFFER_SIZE: (u16, u16);
 
+    /// Delay after lcd reset.
+    const LCD_RESET_DELAY: u32 = 10;
```

Comment:
Let's use something a bit more descriptive for this:
```suggestion
    /// Duration of the active low reset pulse in µs.
    const RESET_DURATION: u32 = 10;
```

Reply:
Fixed